h/db/__init__.py

```diff
@@ -110,7 +110,21 @@ def close_the_sqlalchemy_session(request):
         changes = tracker.uncommitted_changes() if tracker else []
         if changes:
             msg = "closing a session with uncommitted changes %s"
-            log.warning(msg, changes, extra={"stack": True, "changes": changes})
+            log.error(msg, changes, extra={"stack": True, "changes": changes})
```

Comment:
This should never be happening anymore so I've promoted it from a warning to an error (which gets reported to Sentry)